include/cantera/transport/HighPressureGasTransport.h

```diff
@@ -29,35 +62,48 @@ namespace Cantera
  *     https://github.com/Cantera/cantera/issues/267 for additional information.
  *
  * The implementation employs a method of corresponding states, using the Takahashi
- * @cite takahashi1975 approach for binary diffusion coefficients (using multicomponent
- * averaging rules for the mixture properties), and the Lucas method for the viscosity
- * of a high-pressure gas mixture. All methods are described in Poling et al.
+ * @cite takahashi1975 approach for binary diffusion coefficients (using mixture
+ * averaging rules for the mixture properties), the Lucas method for the viscosity, and
+ * a method from Ely and Hanley. All methods are described in Poling et al.
  * @cite poling2001 (viscosity in Ch. 9, thermal conductivity in Ch. 10, and diffusion
  * coefficients in Ch. 11).
  *
  * @ingroup tranprops
  */
-class HighPressureGasTransport : public MultiTransport
+class HighPressureGasTransport : public MixTransport
 {
 protected:
     //! default constructor
     HighPressureGasTransport() = default;
 
 public:
     string transportModel() const override {
-        return "HighPressureGas";
+        return "high-pressure";
     }
 
-    //! Return the thermal diffusion coefficients (kg/m/s)
-    /*!
-     *  Currently not implemented for this model
+    /**
+     * Returns the mixture high-pressure thermal conductivity in W/m/K
+     * a method by Ely and Hanley.
```

Comment:
```suggestion
     * Returns the mixture high-pressure thermal conductivity in W/m/K
     * using a method by Ely and Hanley.
```
Is there a relevant equation/section to reference from the Poling book for this?

Reply:
This method does not seem to be from the Poling book as far as I can tell. I still haven't found a paper or reference for where they got this implementation from.